git_repo/repo.py

```diff
@@ -163,15 +163,16 @@ def _guess_repo_slug(self, repository, service, resolve_targets=None):
         for remote in repository.remotes:
             if remote.name in targets:
                 for url in remote.urls:
-                    if url.startswith('https'):
-                        if url.endswith('.git'):
-                            url = url[:-4]
+                    if url.endswith('.git'):
+                        url = url[:-4]
+                    if url.startswith('https') or url.startswith('http'):
                         *_, user, name = url.split('/')
                         self.set_repo_slug('/'.join([user, name]))
                         return
+                    elif url.startswith('ssh://'):
```

Comment:
> *hmm... rebase this PR's branch onto new one?*

because your commits are already in order, actually you don't need to go into that much trouble. 

Just do:

```
% git checkout 460d27e19bf322c51eb3d7e03c53458830074b9c
% git co -b pr-sshpatch
% git push github pr-sshpatch # or your origin branch
% git hub request guyzmo/git-repo create # or click in the webui
```